### DISCUSSION TITLE
Slider for selecting timeline data - under the chart

### DISCUSSION THREAD

**Discussion by petrvecera** [plouc/nivo#2231] [Q&A] [answered]
Hi guys

Does Nivo support some build in slider? 
I can't find out how it's properly named, but I mean the slider to select zoom / select  the range of data.

Something like this
![image](https://user-images.githubusercontent.com/8086995/215621676-93013e23-ff5a-49d7-9d0b-25ed49aec2a1.png)

https://charts.ant.design/en/examples/area/stacked#basic-slider

Thanks a lot

**Comment by plouc** [comment, marked as answer]
Hi @petrvecera, no it doesn't.

**Comment by SenouLynn** [comment]
I was able to get around this in React by wrapping the Chart rendering logic inside a Context which holds the data that gets displayed. You can use a generic slider component from MUI or Bootstrap, pass it range parameters then update the the Context state onChange to mutate the range of data passed to the displayed chart. This really only works with longitudinal data. If people are interested I can throw up some code.

**Reply by tiansial** [reply]
@SenouLynn that would be amazing
